git_machete/cli.py

```diff
@@ -556,6 +563,14 @@ def launch(orig_args: List[str]) -> None:
                     machete_client.annotate(branch, parsed_cli.annotation_text)
                 else:
                     machete_client.print_annotation(branch)
+        elif cmd == "clean":
+            machete_client.read_definition_file()
+            machete_client.delete_untracked(opt_yes=cli_opts.opt_yes)
+            machete_client.delete_unmanaged(opt_yes=cli_opts.opt_yes)
+            machete_client.delete_unmanaged(opt_yes=cli_opts.opt_yes)
+            if 'checkout_my_github_prs' in parsed_cli:
+                machete_client.checkout_github_prs(pr_nos=[],
+                                                   my_opened_prs=True)
```

Comment:
Also! would be good to add empty lines b/w the sections:

![image](https://user-images.githubusercontent.com/3383210/156632278-1194fde5-f4a5-4e1d-9a20-db92e783dffd.png)

Reply:
@PawelLipski I don't think it is needed, since we print main section titles in bold and also `delete unmanage` can be called on their own and then, to not put new line would require extra logic which i find unnecessary, wdyt?

Reply:
Okay, maybe leave as is indeed

Reply:
Or maybe... print the headers directly here in cli.py? instead of in client.py 🤔 

Note that there is still no header for checking out current user's PRs, which would also be useful

Reply:
Printing directly in cli.py seems like a less elegant solution (i would have to repeat the same things twice), but i will add the header for checking out current user's PRs